docs/site/Rest-Crud-generator.md

```diff
@@ -0,0 +1,76 @@
+---
+lang: en
+title: 'REST CRUD model endpoints generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Rest-Crud-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+Adds a new
+[Rest config for model endpoint (or multiple backed by the same datasource)](Creating-CRUD-REST-apis.md)
+to a LoopBack application with one single command.
+
+```sh
+lb4 rest-crud [options]
+```
+
+### Options
```

Comment:
Can there also be an option to customize the basepath?

Reply:
We allow generation of multiple configs at once. It will be difficult to prompt for `basePath` so many times. The `basePath` can be customized in `src/model-endpoints/*.ts`.

Reply:
I see, thank you!

Reply:
I added a prompt if only model is selected.